### PR TITLE
chore #164730957 Implementation of twilio

### PR DESCRIPTION
**Pull request by beejay1293**
### What does this PR do?

This chore implements the use of twilio to deliver messages via SMS

### Description of the task to be completed?

 * Twilio Setup and Implementation

### How should this be tested manually?
 * by visiting the link below in V2
  https://andela-epic-mail.herokuapp.com/

### What are the relevant pivotal tracker stories?

<a href="https://www.pivotaltracker.com/story/show/164730957">#164730957<a/>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,11 @@ EPIC Mail is an Application that helps people exchange messages/information over
 - Admin/Moderator can **delete a user from a group**
 - Users can **send an email to a group**
 
+## ROptional Features
```
Comment (CEOehis) [inline README.md]: Typo

File: server/controllers/MessageController.js [hunk 2]
```diff
@@ -169,6 +176,13 @@ class MessageController {
       const inbox = [receiver.rows[0].id, rows[0].id];
       await DB.query(inboxQueryString, inbox);
       const msg = rows[0];
+
+      // deliever messages with twilio
+      client.messages.create({
+        to: '+2348169504447',
```
Comment (CEOehis) [inline server/controllers/MessageController.js]: Is there a reason why these numbers are hardcoded?

**Review by CEOehis — CHANGES_REQUESTED** [review]

File: server/controllers/MessageController.js (hunk 2 above)
Comment (beejay1293) [reply server/controllers/MessageController.js]: No reason, Would it be okay if I set the number as an environment variable in .env file
Would resolve the typo issue in my README.

File: README.md (hunk 1 above)
Comment (beejay1293) [reply README.md]: Would correct that now

File: server/controllers/MessageController.js (hunk 2 above)
Comment (CEOehis) [reply server/controllers/MessageController.js]: Yes. The number should be in the `.env` file

**Pull request merged**